llpc/util/llpcElfWriter.cpp

```diff
@@ -735,6 +737,282 @@ template <class Elf> void ElfWriter<Elf>::updateMetaNote(Context *pContext, cons
   pNewNote->data = data;
 }
 
+// =====================================================================================================================
+// Get the count of relocs in the reloc section
+//
+// @param relocSection : Reloc section
+template <class Elf> size_t ElfWriter<Elf>::numRelocs(const SectionBuffer *relocSection) {
```

Comment:
missing documentation line for `@returns`

Reply:
Fixed.